generator-templates/webclient/EntityDetail.ftl

```diff
@@ -18,8 +18,7 @@ import {
     entityDefault,
     entityFields
 } from "../../modifiers/${entity.name}Modifier";
-import EntityDetail from "../../commons/form/EntityDetail";
-import {addSelectLists} from "../../modifiers/DefaultModifier";
+import {EntityDetail, addSelectLists} from "@starwit/react-starwit";
```

Comment:
Ist addSlelectLists da drinne? Ist das passend in react-starwit?

Reply:
jepp

Reply:
sobald ich es noch einmal bauen kann